website/content/docs/about-vault/how-vault-works.mdx

```diff
@@ -0,0 +1,121 @@
+---
+layout: docs
+page_title: How Vault works
+description: >-
+  Learn about the internal architecture of Vault.
+---
+
+# How Vault works
+
+
+Vault is an identity-based secrets and encryption management system
+that centralizes secret management, rotates old credentials, generates
+credentials on demand, audits client interactions, and supports regulatory
+compliance.
+
+It provides encryption services that are gated by authentication and authorization 
+methods to ensure secure, auditable and restricted access to _secrets_. 
+
+A secret is anything that you want to tightly control access to, such as tokens,
+API keys, passwords, encryption keys or certificates. Vault provides a unified
+interface to any secret, while providing tight access control and recording a
+detailed audit log.
+
+API keys for external services, credentials for service-oriented architecture
+communication, etc. It can be difficult to understand who is accessing which
+secrets, especially since this can be platform-specific. Adding on key rolling,
+secure storage, and detailed audit logs is almost impossible without a custom
+solution. This is where Vault steps in.
+
+Vault validates and authorizes clients (users, machines, apps) before providing
+them access to secrets or stored sensitive data.
+
+![How Vault Works](/img/how-vault-works.png)
```

Comment:
FYI:  The diagram [here](https://drive.google.com/file/d/1mLL5oStjjniIbKhtq0M4x9JH-7Cl4hyH/view?usp=sharing) looks more up-to-date (using the new logo and all) except it looks like a screenshot of a presentation.

Reply:
I'm fine with using it for now since it's more useful than the current image.